pkg/sm/plugins/ufm/ufm_test.go

```diff
@@ -73,7 +73,7 @@ var _ = Describe("Ufm Subnet Manager Client plugin", func() {
 			plugin := &ufmPlugin{client: client, conf: UFMConfig{}}
 			err := plugin.Validate()
 			Expect(err).To(HaveOccurred())
-			Expect(err.Error()).To(Equal("validate(): failed to connect to fum subnet manger: failed"))
+			Expect(err.Error()).To(Equal("failed to connect to ufm subnet manager: failed"))
```

Comment:
why is it needed to validate strings in an error object ?
@moshe010 is this something that was required ?

Reply:
@adrianchiris It is just more check for the error

Reply:
just to make sure you failing on the right error. In c++ I would check the type of exception ...

Reply:
i dont think this should be checked. Error occured is enough. 
or at least move all the error string to their own file and define as consts then use it.

Reply:
anyway thats just my opinion on the matter... so i understand it was required to check in the  test

Reply:
we could also extend the error interface with a "code" attribute and implement an Errorf() of our own to decouple the string from the type of error that occured, just a thought ...

Reply:
I will look what other open source project will do,but for this PR let keep it as is.